### PR TITLE
AC: Added field for normalization arguments

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/config/config_validator.py [hunk 1]
```diff
@@ -568,6 +568,47 @@ def parameters(self):
...
+        super().__init__(**kwargs)
+
+    def validate(self, entry, field_uri=None, fetch_only=False, validation_scheme=None):
+        error_stack = super().validate(entry, field_uri, fetch_only, validation_scheme)
+        if entry is None:
+            return error_stack
+
+        field_uri = field_uri or self.field_uri
+
+        precomputed_args = NormalizationArgsField.PRECOMPUTED_MEANS if 'mean' in field_uri else (
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/config/config_validator.py:592]: precomputed values can be one from __init__ args, no need to use field_uri as hint

File: tools/accuracy_checker/accuracy_checker/config/config_validator.py [hunk 2]
```diff
@@ -568,6 +568,47 @@ def parameters(self):
...
+            NormalizationArgsField.PRECOMPUTED_STDS)
+
+        entry = get_or_parse_value(entry, precomputed_args)
+
+        if 'std' in field_uri and 0 in entry:
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/config/config_validator.py:597]: also more general way, if you add parameter allow_zeros=True/False

File: tools/accuracy_checker/accuracy_checker/config/config_validator.py [hunk 3]
```diff
@@ -568,6 +568,47 @@ def parameters(self):
...
+            reason = "{} should not contain 0".format(field_uri)
+            if not fetch_only:
+                self.raise_error(entry, field_uri, reason)
+            error_stack.append(self.build_error(entry, field_uri, reason, validation_scheme=validation_scheme))
+
+        if not (len(entry) == 3 or len(entry) == 1):
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/config/config_validator.py:603]: num_channels parameter (we have also 3d case, where number of channels different)

**Comment by Anna-Mironova** [comment]
Jenkins please retry a build

**Comment by Anna-Mironova** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/accuracy_checker/config/config_validator.py (hunk 1 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/config/config_validator.py]: Added

File: tools/accuracy_checker/accuracy_checker/config/config_validator.py (hunk 2 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/config/config_validator.py]: Added

File: tools/accuracy_checker/accuracy_checker/config/config_validator.py (hunk 3 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/config/config_validator.py]: Added

**Review by eaidova — APPROVED** [review]

**Pull request merged**